profiles/desc/curl_default_ssl.desc

```diff
@@ -0,0 +1,11 @@
+# Copyright 1999-2020 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+# This file contains descriptions of CURL_DEFAULT_SSL USE_EXPAND flags for net-misc/curl
+
+gnutls - Use GnuTLS
```

Comment:
These should explain what happens when you set them, i.e. what's the diff between 'default' and 'multi' target thingie.